### DISCUSSION TITLE
Suggest to truncate current time for both auto-generated fields created_at/updated_at

### DISCUSSION THREAD

**Discussion by vuon9** [aarondl/sqlboiler#1011] [General]
Hi, I suggest changing a bit in behavior of generating `currTime` for both `created_at` `updated_at` fields. They're now just using `time.Now` without truncate, so the data after marshal to JSON will also contain the unit value less than second, and I think it shouldn't because of inconsistency with result data from DB selecting. The DB data we query via sqlboiler always return the value for date time field as second rounded.

Current:
```go
currTime := time.Now().In(boil.GetLocation())
```

Suggest to change to:
```go
currTime := time.Now().In(boil.GetLocation()).Truncate(time.Second)
```

But I'm not sure if there is really valid for other SQL language, my experience is just about mysql, please share. I can make PR as well.

Thanks.

**Comment by aarondl** [comment]
> and I think it shouldn't because of inconsistency with result data from DB selecting. The DB data we query via sqlboiler always return the value for date time field as second rounded.

This seems very specific to your use case. Some people may appreciate the additional precision. Is this causing some sort of issue for you? Or is it just that you don't want your users seeing a time with so much precision?

**Comment by vuon9** [comment]
> Or is it just that you don't want your users seeing a time with so much precision?

Hi, it's about this one.
